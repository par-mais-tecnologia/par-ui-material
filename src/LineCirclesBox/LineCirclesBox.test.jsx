import React from 'react'
import { configure, shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import { LineCirclesBox } from '../index'

configure({ adapter: new Adapter() })

describe('LineCirclesBox', () => {
  it('shallow LineCirclesBox', () => {
    const wrapper = shallow(
      <LineCirclesBox lineColor='#991d99'>
        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
      </LineCirclesBox>
    )
    expect(wrapper).toBeTruthy()
  })

  it('shallow LineCirclesBox match Snapshot', () => {
    const wrapper = shallow(
      <LineCirclesBox lineColor='#99251d'>
        Lorem ipsum dolor sit amet.
      </LineCirclesBox>
    )
    expect(wrapper).toMatchSnapshot()
  })
})
