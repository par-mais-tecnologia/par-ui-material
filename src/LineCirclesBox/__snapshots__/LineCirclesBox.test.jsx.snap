// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LineCirclesBox shallow LineCirclesBox match Snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <WithStyles(LineCirclesBox)
    lineColor="#99251d"
  >
    Lorem ipsum dolor sit amet.
  </WithStyles(LineCirclesBox)>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": "Lorem ipsum dolor sit amet.",
      "classes": Object {
        "line": "LineCirclesBox-line-1",
      },
      "lineColor": "#99251d",
    },
    "ref": null,
    "rendered": "Lorem ipsum dolor sit amet.",
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": "Lorem ipsum dolor sit amet.",
        "classes": Object {
          "line": "LineCirclesBox-line-1",
        },
        "lineColor": "#99251d",
      },
      "ref": null,
      "rendered": "Lorem ipsum dolor sit amet.",
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
